database-commons/src/main/java/io/cdap/plugin/db/connector/AbstractDBSpecificConnectorConfig.java

```diff
@@ -55,4 +63,11 @@ public int getPort() {
   public boolean canConnect() {
     return super.canConnect() && !containsMacro(ConnectionConfig.HOST) && !containsMacro(ConnectionConfig.PORT);
   }
+
+  public String getTransactionIsolationLevel() {
+    if (transactionIsolationLevel == null) {
+      transactionIsolationLevel = TransactionIsolationLevel.Level.TRANSACTION_READ_COMMITTED.name();
```

Comment:
in other plugins we dont have a default isolation level set by us as it can differ db wise, return null if it is null